### ISSUE TITLE
Convert to JSON?

### ISSUE THREAD

**Issue by bebraw** [karlhorky/awesome-speakers#10]
If you do that, it’s easier to use the data for sites etc.

**Comment by robinpokorny** [comment]
Then it would either A) not be readable in the README or B) README would need to be generated. The other awesome lists do not have it as JSON.

**Comment by bebraw** [comment]
Sure. Or extract JSON from README. Easy too.

**Comment by karlhorky** [comment]
Would be cool to be more interop-friendly. Want two things though:

- should appear when you come to the repo page, just like other [Awesome](https://github.com/sindresorhus/awesome) lists (GitHub supports only Markdown files, right?)
- want to keep it easy to update (Markdown is already too hard because of those weird two trailing spaces, IMO)

Wonder if something could achieve both of those things at once. Like a website with a form to CRUD the list where it auto-generates pull requests to modify both the Markdown and the JSON file.

**Comment by karlhorky** [comment]
> Sure. Or extract JSON from README. Easy too.

Yeah, this was my first idea. Have some kind of script or parser (needs to be fairly forgiving, probably) that can generate JSON from the readme.

**Comment by robinpokorny** [comment]
You can include HTML comments in Markdown. That could help a bit.

However, do you really think someone will want to have this data in JSON? And if yes, for what purpose?

**Comment by bebraw** [comment]
@robinpokorny Site (could be in this repo even). Also handy for conf organizers but of course you still have to transform the data to fit your system.

**Comment by karlhorky** [comment]
@stojanovic expressed some interest in helping with this one too just now.

**Comment by stojanovic** [comment]
Yes, idea is to have API endpoint and AWS Lambda function that will convert list to JSON format.

**Comment by erikras** [comment]
Guys, we need a database with a GraphQL API endpoint, with foreign keys to events each person has spoken at, some animated UI, and a mobile app. 💥

Or, you know, just a document. 🤷‍♂️

**Comment by nezdemkovski** [comment]
Hey guys, I created https://confcitizens.com/ which is based on `awesome-speakers` data, but I've already started and it’s still in progress to add more information about speakers (including past/future talks and so on). 

Everything is converted to JSON now 
(https://github.com/yakovlevyuri/confcitizens/blob/master/data/speakers.ts)

And the website is open sourced ofc https://github.com/yakovlevyuri/confcitizens. Want to hear your feedback and feel free to contribute!

**Comment by karlhorky** [comment]
@yakovlevyuri Added you on Twitter. Let's get in touch and see what we can do.
